lib/transforms/base-js.js

```diff
@@ -4,7 +4,7 @@ const path = require('path');
 const handlebarsConfig = {
 	debug: false, // set to true to debug finding partial/helper issues
 	extensions: ['.html'],
-	helperDirs: [path.resolve('./node_modules/@financial-times/n-handlebars/src/helpers'), path.resolve('./server/helpers')],
+	helperDirs: [path.resolve('./node_modules/@financial-times/n-handlebars/src/helpers'), path.resolve('./server/helpers'), path.resolve('./node_modules/@financial-times/n-image/src/presenters')],
```

Comment:
@debugwand they do get picked up, not ignored... so probably good idea to put it in a separate folder